P202.Training.Test/RoleServiceTest.cs

```diff
@@ -0,0 +1,128 @@
+﻿using AutoMapper;
+using Moq;
+using P202.Training.Data.Entities;
+using P202.Training.Data.Repositories;
+using P202.Training.Domain;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace P202.Training.Test
+{
+    public class RoleServiceTest
+    {
+        [Fact]
+        public void RoleService_CreateData()
+        {
+            // Arrange
+            Mock<IRoleRepository> roleRepository = new Mock<IRoleRepository>();
+            Mock<IMapper> mappingService = new Mock<IMapper>();
+
+            var newRole = new Role
+            {
+                Id = 1,
+                Name = "Admin"
+            };
+
+            roleRepository.Setup(r => r.CreateRole(newRole));           
```

Comment:
There are lots of spaces added at the end of the line, please check those are removed everywhere

Reply:
Fixed!